packages/jsii-pacmak/lib/targets/sphinx.ts

```diff
@@ -651,9 +651,14 @@ class SphinxDocsGenerator extends Generator {
             this.code.line();
         }
         const readonly = prop.immutable ? ' *(readonly)*' : '';
-        const abs = prop.abstract ? ' *(abstract)*' : '';
+        const abs = (parent.kind !== spec.TypeKind.Interface &&  prop.abstract) ? ' *(abstract)*' : '';
```

Comment:
I wonder about the value of showing "abstract" at all.

Does it help consumers of a class know that a method is abstract? Or do they simply care about what it would do if they were to call it?

Reply:
I think they do if they are looking to extend a class.